nipy/labs/viz_tools/activation_maps.py

```diff
@@ -342,6 +342,8 @@ def plot_anat(anat=None, anat_affine=None, cut_coords=None, slicer='ortho',
             you whish to save figures with a black background, you
             will need to pass "facecolor='k', edgecolor='k'" to pylab's
             savefig.
+        dim: float, optional
+            If set, dim the anatomical image.
```

Comment:
no, I believe it selects the maximum value for plotting to be sth from the mean above
```
vmax = vmean + (1+dim)*ptp
```

I didn't know how to phrase it precisely